templates/bundles/DemosPlanCoreBundle/DemosPlanAssessmentTable/DemosPlan/dhtml/v1/assessment_table_new_statement.html.twig

```diff
@@ -14,7 +14,12 @@
     } %}
 
     {% set form_action = templateVars.formAction|default(path("DemosPlan_statement_new_submitted", {'procedureId': templateVars.table.procedure.ident})) %}
-    {% set abort_path = templateVars.abortPath|default(path("dplan_assessmenttable_view_original_table", { 'procedureId': templateVars.table.procedure.ident })) %}
+    {% if hasPermission('area_statement_data_input_orga') %}
```

Comment:
If the user is "Datenerfasser*in", should it always be this route or might it also be defined via `templateVars.abortPath` (so that it should be
```
{% set abort_path = templateVars.abortPath|default(path('DemosPlan_statement_orga_list', { 'procedureId': templateVars.table.procedure.ident })) %}
```
)?